conf/nftables/nftables.d/fail2ban.conf

```diff
@@ -0,0 +1 @@
+# FIXME:
```

Comment:
?

Reply:
that means: i'm not sure it does what I think it does, we should test that.

Reply:
I tested, it works.
It adds this in the ruleset:
```
table inet f2b-table {
	set addr-set-sshd {
		type ipv4_addr
		elements = { 10.248.215.1 }   <---------------- the ip of the host of the container that tried too many ssh auth.
	}

	chain INPUT {
		type filter hook input priority filter - 1; policy accept;
		tcp dport 22 ip saddr @addr-set-sshd reject with icmp port-unreachable
	}
}
```
unbanip works as expected too.